com.varnamproject.Varnam.json

```
@@ -0,0 +1,171 @@
+{
+    "app-id": "com.varnamproject.Varnam",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
```

Update runtime to 19.08
```suggestion
    "runtime-version": "19.08",
```